malduck/procmem/procmem.py

```diff
@@ -710,10 +841,17 @@ def _findbytes(self, yara_fn, query, addr, length):
         else:
             return []
 
-    def findbytesp(self, query, offset=0, length=None):
+    def findbytesp(
+        self,
+        query: Union[str, bytes],
```

Comment:
Probably `query: bytes` is intended here?
There is probably some call somewhere to `findbytesp` with str as an argument

Reply:
Not sure. Query is usually provided as literal e.g. `p.findbytesp("22 33 44 55")` so can be both string and bytes.